Classes/GraphQL/Resolver/Type/MutationResolver.php

```diff
@@ -488,6 +488,34 @@ public function deleteAssetCollection($_, array $variables): array
         ];
     }
 
+    /**
+     * @param $_
+     * @param array $variables
+     * @return AssetCollection|null
+     * @throws Exception
+     */
+    public function updateAssetCollection($_, array $variables): AssetCollection
```

Comment:
```suggestion
    public function updateAssetCollection($_, array $variables): ?AssetCollection
```

Reply:
What does question mark mean here? Sorry, I'm a graphql noob.

Reply:
This is PHP related and just means that `null` is a possible return value. Like you already have in the PHPDoc comment.

Reply:
LOL, my brain had thought it was looking at graphql definition 😄